packages/edit-site/src/components/dataviews/view-list.js

```diff
@@ -49,7 +52,8 @@ const sortingItemsInfo = {
 	desc: { icon: arrowDown, label: __( 'Sort descending' ) },
 };
 const sortIcons = { asc: chevronUp, desc: chevronDown };
-function HeaderMenu( { dataView, header } ) {
+// TODO: why doesn't Header work with view/onChangeView?
+function HeaderMenu( { dataView, header, view, onChangeView } ) {
```

Comment:
I've noticed the `HeaderMenu` doesn't take the `fields`, `view`, `onChangeView` as properties, but uses tanstack instead. Can we implement this component's functionality by only using our own abstractions?

Having to use two things for the same operations is confusing. For the demo, I chose to just pass `view` and `onChangeView`, but we need to decide how this works.

Reply:
Set to use the existing `dataView`, `header`. If this needs change, it'd be done in a different PR.